### PR TITLE
Less ambiguous wording for 'topology.objects' value.

### PR DESCRIPTION
**Pull request by vicapow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ TopoJSON always consists of a single topology object. A topology may contain any
 
 A topology is a TopoJSON object where the type member's value is “Topology”.
 
-  * A topology must have a member with the name “objects” whose value is another object, each of which is a named geometry object.
+  * A topology must have a member with the name “objects” whose value is another object, whose each property is a geometry object.
```
Comment (calvinmetcalf) [inline README.md]: this should probably be still `named geometry object`

Comment (vicapow) [inline README.md]: I guess the spec also has: https://github.com/mbostock/topojson-specification#215-objects

> 2.1.5. Objects
> A topology must have an “objects” member whose value is an object. This object may have any number of members, whose value must be a geometry object.

so it's not really _that_ ambiguous.

Comment (mbostock) [inline README.md]: Okay, I pushed some new wording. Let me know if that’s better or send another PR with further edits. Thanks!

**Comment by vicapow** [comment]
Thanks, Mike!
